supervisor/backups/backup.py

```diff
@@ -58,6 +58,7 @@
         "*.db-shm",
         "__pycache__/*",
         "*.log",
+        "*.log.*",
```

Comment:
Why are we even excluding the log in the first place?
That is a part of the configuration directory?

Reply:
I guess they were not considered valuable enough to preserve? I kinda agree with that. If you restore an old backup on the same device, the logs are still there (and its probably more likely that you want to look at the last logs instead of the logs from the backup, but that is speculation and depends on the situation).

Reply:
In any case, the current situation seems incoherent. Either we add this line, or we remove the one above. 

@pvizeli thoughts?

Reply:
My vote is removing the current line

Reply:
I have zero opinion on that

Reply:
I really think there are zero benefits of having them backupped.
I would exclude them all.